cloudify_cli/commands/logs.py

```diff
@@ -110,7 +116,11 @@ def download(output_path, all_nodes, logger):
                 conn,
                 CloudifyNodeType.MANAGER,
                 output_path,
-                logger)
+                logger,
+                output_json)
+
+    if get_global_json_output():
+        output(json.dumps(output_json))
```

Comment:
So your system test would parse the stdout of the function?

Reply:
You might want to use the `cls` parameter like here https://github.com/cloudify-cosmo/cloudify-cli/blob/master/cloudify_cli/commands/cluster.py#L173